backend/src/lib/googleOAuth.ts

```diff
@@ -0,0 +1,67 @@
+import { OAuth2Client } from 'google-auth-library'
+import { GOOGLE_CLIENT_ID, GOOGLE_CLIENT_SECRET, GOOGLE_OAUTH_REDIRECT_URI } from '@/settings'
+
+export interface GoogleUserInfo {
+    sub: string
+    email: string
+    email_verified: boolean
+    name: string
+    given_name: string
+    family_name: string
+    picture: string
+}
+
+export class GoogleOAuthService {
```

Comment:
yet another "service". does this need to be a "service"? also if it's a service shouldn't it be inside services/ ?

I'm not the biggest fan of this everything is a "service" model. this class barely holds any state and I think we can just have client be a global because the params never change. we're currently initializing this class on every request for no reason. 

I think these should just be util functions